src/Orleans/Configuration/Legacy/LegacyConfigurationExtensions.cs

```diff
@@ -30,10 +28,18 @@ public static IServiceCollection AddLegacyClusterConfigurationSupport(this IServ
             return services;
         }
 
-        public static IServiceCollection AddLegacyClientConfigurationSupport(this IServiceCollection services)
+        public static IServiceCollection AddLegacyNodeConfigurationSupport(this IServiceCollection services)
```

Comment:
BTW, these legacy transformation methods should be called automatically when loading the legacy configuration into SiloBuilder and ClientBuilder.

Reply:
nevermind, I now saw that at least in DefaultSiloServices you are calling this one. I assume it's the same for others, so please ignore

Reply:
As we don't really want legacy to be the primary usage, I'm wondering if we want to put legacy hosting options into it's own namespace, so users need to explicitly opt into it to use it.  This would make the new and expected patterns more clear and not confuse users of the new experience with legacy support.
Maybe Orleans.Hosting.Legacy?
We can also make this decision at a later time, once the new hosting patterns have baked a bit more.

Reply:
I suggest that for Beta have everything in one place, after the FULL options revamp we should move types around (Orleans.Core, Orleans.Runtime), namespaces, etc.
I don't think we've Legacy Hosting ;-) so that's maybe not the best. My plan is that after the full Options work we'll be able to move all legacy hosting options along with the types out of Orleans assemblies to some Legacy ones.